gpu-service/main.py

```
@@ -0,0 +1,145 @@
+try:
+    qdrant.get_collection(collection_name='faces')
+    logger.info("Collection 'faces' already exists.")
+except Exception:
+    logger.info("Collection 'faces' not found, creating it.")
+    qdrant.create_collection(
+        collection_name='faces',
+        vectors_config=models.VectorParams(size=128, distance=models.Distance.COSINE)
+    )
+
+@app.route('/lookup', methods=['POST'])
+def lookup():
```

Ruff (0.11.9)

Missing return type annotation for public function `lookup`

(ANN201)